### PR TITLE
fix: Update Agent Builder branding. Modularize install. Update documentation. 

### PR DESCRIPTION
**Pull request by paulramsey**
# Description

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [X] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [X] You are listed as the author in your notebook or README file.
  - [X] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [X] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [X] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [X] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: gemini/sample-apps/genwealth/deployment/enable-apis.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Enable Backend APIs
+PROJECT_ID=$(gcloud config get-value project 2> /dev/null)
+gcloud services enable iam.googleapis.com --project "${PROJECT_ID}"
```
Comment (holtskinner) [inline gemini/sample-apps/genwealth/deployment/enable-apis.sh:5]: You might be able to put all of these in a single command.

**Review by holtskinner — APPROVED** [review]

File: gemini/sample-apps/genwealth/deployment/enable-apis.sh (hunk 1 above)
Comment (paulramsey) [reply gemini/sample-apps/genwealth/deployment/enable-apis.sh]: Looks like it will only accept up to 20 per batch, so I combined this into two batches.

**Comment by paulramsey** [comment]
@holtskinner - Looks like "OwlBot Post Processor" has been stuck since yesterday. Is there anything I can do to nudge that process?

**Comment by holtskinner** [comment]
> @holtskinner - Looks like "OwlBot Post Processor" has been stuck since yesterday. Is there anything I can do to nudge that process?

You can add the `owlbot:run` tag. It's required for all forks

**Pull request merged**